x-pack/plugins/apm/public/hooks/use_preferred_service_anomaly_timeseries.ts

```diff
@@ -5,12 +5,43 @@
  * 2.0.
  */
 
+import { ENVIRONMENT_ALL } from '../../common/environment_filter_values';
 import { ApmMlDetectorType } from '../../common/anomaly_detection/apm_ml_detectors';
-import { getPreferredServiceAnomalyTimeseries } from '../../common/anomaly_detection/get_preferred_service_anomaly_timeseries';
+import { Environment } from '../../common/environment_rt';
+import { ServiceAnomalyTimeseries } from '../../common/anomaly_detection/service_anomaly_timeseries';
 import { useApmServiceContext } from '../context/apm_service/use_apm_service_context';
 import { useEnvironmentsContext } from '../context/environments_context/use_environments_context';
 import { useServiceAnomalyTimeseriesContext } from '../context/service_anomaly_timeseries/use_service_anomaly_timeseries_context';
 
+export function getPreferredServiceAnomalyTimeseries({
```

Comment:
if this is exported, can it be a separate file?

Reply:
separeted: https://github.com/elastic/kibana/pull/126679/commits/2d8f66c0e908796e4e4cbfd835ccd3eb1df84eed